CODING_STYLE.md

```diff
@@ -0,0 +1,172 @@
+# Zilliqa Coding Style
+<!--Source: https://github.com/ethereum/aleth/blob/master/CODING_STYLE.md -->
+## Code Formatting
+
+Use clang-format tool to format your changes, see [CONTRIBUTING](CONTRIBUTING.md) for details.
+
+
+## Namespaces
+
+1. No `using namespace` declarations in header files.
+2. Preprocessor symbols should be prefixed with the namespace in all-caps and an underscore.
```

Comment:
The example immediately below item 2 is for item 1.
Maybe move the example up, and add a second example for item 2.